package.json

```diff
@@ -37,7 +37,7 @@
     "react": "^16.5.0",
     "react-aria-modal": "^4.0.0",
     "react-dom": "^16.4.2",
-    "react-router-dom": "^4.3.1",
+    "react-router-dom": "^5.1.0",
```

Comment:
This no longer the latest v5. Would you please update?

Reply:
I did a new commit with react-router-dom upgraded to v5.1.2 and also changed the commit message to fixes #411